packages/functional-tests/pages/login.ts

```diff
@@ -50,6 +50,9 @@ export const selectors = {
   NOT_EMAIL_MET: '#password-same-as-email.password-strength-met',
   NOT_EMAIL_FAIL: '#password-same-as-email.password-strength-fail',
   PERMISSION_ACCEPT: '#accept',
+  CWTS_HEADER: 'text="Choose what to sync"',
```

Comment:
If we start using text selectors here where we can, it's less work when we get to the React conversion work.